ML-Transformers/BackgroundEnchantments/CHANGELOG.md

```diff
@@ -6,8 +6,29 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 The versioning of this project is directly tied to the version of @vonage/ml-transformers dependency
 
+## 5.2.0-alpha.3 - 2024-11-07
+
+### Changed
+
+- Upgrade @vonage/ml-transformers version 5.2.0-alpha.3. which includes the following:
+  - Improved rendering pipeline when in webGL when using WebGL PRECISE renderingOptions
+
+    ```ts
+    renderingOptions: {
+      type: RenderingType.WEBGL,
+      selfieSegmentationType: WebglSelfieSegmentationType.PRECISE
+    }
+    ```
+
+### Fixed
```

Comment:
Do we want to mention the crash fix for the issue we had about changing the renderer pipeline multiple times?

Reply:
Yes good point, adding it